atlas/atlas.sh

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+readonly ATLAS_VERSION='1.1.0'
+readonly ATLAS_FILE="apache-atlas-${ATLAS_VERSION}-bin"
+readonly ATLAS_GCS_URL="gs://dariusz-init-actions/atlas/${ATLAS_FILE}.tar.gz" # TODO(Aniszewski): change to official GCS location before merging
+readonly ATLAS_HOME='/etc/atlas'
```

Comment:
Typically we stores it under /usr/lib/atlas and create a softlink for the conf directory under /etc/atlas.